checks/write.md

```diff
@@ -0,0 +1,33 @@
+# How to write a check
```

Comment:
ignore this file, it will disappear once previous https://github.com/ossf/scorecard/pull/712 and https://github.com/ossf/scorecard/pull/718 are merged